### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by rowild**
- Improve Readme (mention the option to edit config.yaml already in "How to use"; minor grammar fixes)

Done with dedicated [commit](https://github.com/GsTYPO3/ddev-typo3-src/commit/bbdc0401a0a0fbcc8fae74ba093a2f1606033f0c) already:
- Push MariaDB version from 10.3 to 10.5 (10.4 is min requirment for TYPO3 11, which is defined as standard version in config.yaml)

### CODE REVIEW

File: .ddev/config.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ router_https_port: "443"
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
-mariadb_version: "10.3"
+mariadb_version: "10.5"
```
Comment (gilbertsoft) [inline .ddev/config.yaml:10]: Will apply this in a dedicated commit to be able to pick the other changes to all branches.
```suggestion
mariadb_version: "10.3"
```

File: README.md [hunk 2]
```diff
@@ -11,11 +11,12 @@ the post-start hooks which are doing the whole magic.
 
 * Install Docker and DDEV Local (and on Windows also Git)
 * Download and extract [this repository](https://github.com/GsTYPO3/ddev-typo3-src/archive/master.zip)
+* [optional] Edit [installation-folder]/.ddev/config.yaml to your likings
```
Comment (gilbertsoft) [inline README.md:14]:
```suggestion
* [optional] Edit `.ddev/config.yaml` to your likings
```

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gilbertsoft) [reply README.md:14]: @rowild do you think then `[installation-folder]` is important here?

File: .ddev/config.yaml (hunk 1 above)
Comment (gilbertsoft) [reply .ddev/config.yaml]: See https://github.com/GsTYPO3/ddev-typo3-src/commit/bbdc0401a0a0fbcc8fae74ba093a2f1606033f0c

File: README.md (hunk 2 above)
Comment (rowild) [reply README.md:14]: I would mention it, since 
- up to this point the user did not switch to the installation folder; and
- `.ddev` is a hidden folder

**Comment by rowild** [comment]
I am confused: did you change the MariaDB version already before my PR? Did not realise that, sorry!

**Comment by gilbertsoft** [comment]
> 
> 
> I am confused: did you change the MariaDB version already before my PR? Did not realise that, sorry!

Nope, afterwards. See comment https://github.com/GsTYPO3/ddev-typo3-src/pull/3#discussion_r653482582

**Comment by gilbertsoft** [comment]
Thanks!

**Pull request merged**